### DISCUSSION TITLE
请问为什么要用静态链接的 openssl

### DISCUSSION THREAD

**Discussion by linsui** [pot-app/pot-desktop#327] [General]
Linux 发行版打包一般倾向于动态链接系统的 openssl，如果没有特殊原因，我在 NixOS 的包里打个 patch 仍然使用系统的 openssl。

**Comment by Pylogmon** [comment]
现阶段各大发行版的openssl版本不一致，有的没有3.0版本，有的没有1.1版本，静态链接是为了只打一个包适用于所有发行版。

**Reply by linsui** [reply]
那我打包的时候 patch 一下。 :ok:
